mathics/builtin/no_meaning.py

```diff
@@ -26,36 +26,41 @@
 
-for affix, format_fn, operator_base_class in (
-    ("infix", "Infix", NoMeaningInfixOperator),
-    ("postfix", "Postfix", NoMeaningPostfixOperator),
-    ("prefix", "Prefix", NoMeaningPrefixOperator),
-):
-    for operator_name, operator_tuple in OPERATOR_DATA[
-        f"no-meaning-{affix}-operators"
-    ].items():
-        # Create the Mathics3 Builtin class...
-        operator_string = operator_tuple[0]
-        generated_operator_class = type(
-            operator_name,
-            (operator_base_class,),
-            {
-                "__doc__": operator_base_class.__doc_pattern__.format(
-                    operator_name=operator_name, operator_string=operator_string
-                ),
-                "operator": operator_string,
-                "summary_text": f"""{operator_name} {affix} operator "{operator_string}" (no pre-set meaning attached)""",
-                "formats": {
-                    (
-                        ("InputForm", "OutputForm", "StandardForm"),
-                        f"{operator_name}[args__]",
-                    ): (('%s[{args}, "%s"]' % (format_fn, operator_string)))
+
+def init():
+    for affix, format_fn, operator_base_class in (
+        ("infix", "Infix", NoMeaningInfixOperator),
+        ("postfix", "Postfix", NoMeaningPostfixOperator),
+        ("prefix", "Prefix", NoMeaningPrefixOperator),
+    ):
+        for operator_name, operator_tuple in OPERATOR_DATA[
+            f"no-meaning-{affix}-operators"
+        ].items():
+            # Create the Mathics3 Builtin class...
+            operator_string = operator_tuple[0]
+            generated_operator_class = type(
+                operator_name,
+                (operator_base_class,),
+                {
+                    "__doc__": operator_base_class.__doc_pattern__.format(
+                        operator_name=operator_name, operator_string=operator_string
+                    ),
+                    "operator": operator_string,
+                    "summary_text": f"""{operator_name} {affix} operator "{operator_string}" (no pre-set meaning attached)""",
+                    "formats": {
+                        (
+                            ("InputForm", "OutputForm", "StandardForm"),
+                            f"{operator_name}[args__]",
+                        ): (('%s[{args}, "%s"]' % (format_fn, operator_string)))
+                    },
                 },
-            },
-        )
-        if affix == "infix":
-            # FIXME get precedence from JSON. Value 100 is a hack for now.
-            mathics.core.parser.operators.flat_binary_ops[operator_name] = 100
-
-        # Put the newly-created Builtin class inside this module.
-        setattr(modules[__name__], operator_name, generated_operator_class)
+            )
+            if affix == "infix":
+                # FIXME get precedence from JSON. Value 100 is a hack for now.
+                mathics.core.parser.operators.flat_binary_ops[operator_name] = 100
```

Comment:
I forgot that I also need to fix this. Instead of 100, we should use the value in JSON. At the time I wrote that I had not had the precedence values in the JSON yet.

Reply:
And I also need to handle affix's "prefix" and "postfix".

Reply:
And finally, on my todo is to turn "no meaning" into a guide section and have modules underneath for each of the affix's. When you look at this section in Django (or the PDF) it is very long.

Reply:
@rocky, If you want, you can merge this, and then `SYMBOL_MANIFEST.txt` is going to be updated. Or once you fix this, just run the script to update the manifest.